### PR TITLE
Update hold icon

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

shrinks hold icon
# Why

Fixes styling concerns for more button options

https://skype.visualstudio.com/SPOOL/_workitems/edit/2958022
# How Tested

image

### CODE REVIEW

**Comment by anjulgarg** [comment]
Screen Shot 2022-08-25 at 8 41 35 AM
Hold button missing in mobile snapshots

**Review by carocao-msft — APPROVED** [review]

**Review by edwardlee-msft — COMMENTED** [review]
Looks like the icons in the contextual menu all shifted to the right by a smidge.

image

File: packages/communication-react/review/beta/communication-react.api.md [hunk 1]
```diff
@@ -1562,7 +1563,8 @@ export const DEFAULT_COMPONENT_ICONS: {
     ParticipantItemOptions: JSX.Element;
     ParticipantItemOptionsHovered: JSX.Element;
     ParticipantItemScreenShareStart: JSX.Element;
-    HoldCall: JSX.Element;
+    HoldCallContextMenuItem: JSX.Element;
```
Comment (edwardlee-msft) [inline packages/communication-react/review/beta/communication-react.api.md:1566]: nit: Should be renamed to HoldCallContext**ual**MenuItem to be consistent with the other MenuItem names

File: change/@internal-react-composites-4ccc3a12-4ea0-4d59-bb6b-eebea91e015f.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Add Icon for hold button when its in a context menu",
```
Comment (edwardlee-msft) [inline change/@internal-react-composites-4ccc3a12-4ea0-4d59-bb6b-eebea91e015f.json:3]: nit: Change to 
"Center icons and add hold button icon into Contextual Menu"

**Review by edwardlee-msft — APPROVED** [review]

**Pull request merged**